Docs slow and frozen frames Android (#4028)
getsentry/sentry-docs: src/platforms/android/common/performance/instrumentation/automatic-instrumentation.mdx

</manifest>
```

### Slow and Frozen Frames

> Unresponsive UI, animation hitches, or just jank annoy users and degrade the user experience. Two measurements to track this unwanted experience are slow frames and frozen frames. A smoothly running app should try to avoid both. That's why the SDK adds these two measurements for the transactions you capture.

Author: ```suggestion
Unresponsive UI, animation hitches, or just jank annoy users and degrade the user experience. Two measurements to track this unwanted experience are slow frames and frozen frames. A smoothly running app should try to avoid both. That's why the SDK adds these two measurements for the [Android's Activity](/platforms/android/performance/instrumentation/automatic-instrumentation/#androids-activity-instrumentation) transactions.
```

> Unresponsive UI, animation hitches, or just jank annoy users and degrade the user experience. Two measurements to track this unwanted experience are slow frames and frozen frames. A smoothly running app should try to avoid both. That's why the SDK adds these two measurements for the [Android's Activity](/platforms/android/performance/instrumentation/automatic-instrumentation/#androids-activity-instrumentation) transactions.

Reviewer: Let's replace the word "jank" with something more meaningful. I've made a suggested edit to avoid this word, but if there's a better term, add it in.

```suggestion
Unresponsive UI and animation hitches annoy users and degrade the user experience. Two measurements to track these types of experiences are *slow frames* and *frozen frames*. If you want your app to run smoothly, you should try to avoid both. The SDK adds these two measurements for the [Android's Activity](/platforms/android/performance/instrumentation/automatic-instrumentation/#androids-activity-instrumentation) transactions.
```

> - **Slow frames** are captured by the SDK when the app needs more than 16 ms for a frame. Typically, a phone or tablet renders with 60 frames per second (fps). With 60 fps, every frame has 16 ms to render; slower than that, and the app has slow frames.

Reviewer: ```suggestion
- **Slow frames** are captured by the SDK when the app takes more than 16 ms to render a frame. Typically, a phone or tablet renders 60 frames per second (fps). At 60 fps, every frame has 16 ms to render; slower than that, and the app has slow frames.
```
